### PR TITLE
Hub catalog release 2.1.1

### PR DESCRIPTION
**Pull request by rigazilla**
With 2.1.0 Hub release i tried to follow the operator-sdk folder structure, but I think `operator-sdk generate csv` doesn't provide any big advantage for us and has the drawback to generate a new folder for each release making hard to check changes in the PR.

I'm reverting back the new CSV file to the old structure in this PR for 2.1.1

### CODE REVIEW

**Comment by dmvolod** [comment]
@rigazilla probably newer version of the Operator SDK adds some advantage.. We feel free to update at least two next release or newer.

**Comment by rigazilla** [comment]
@dmvolod probably you're right, we should consider to upgrade on master branch: al least we need bundle support

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -32,119 +43,74 @@ metadata:
...
+            "path": "asdasd"
+          }
         }
       ]
+    capabilities: Full Lifecycle
+    categories: Database
+    repository: https://github.com/infinispan/infinispan-operator
     description: Create and manage Infinispan clusters.
-    containerImage: quay.io/infinispan/operator:latest
+    containerImage: quay.io/infinispan/operator:2.0.5.Final
```
Comment (dmvolod) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:68]: @rigazilla is 2.0.5 correct operator image?

Comment (rigazilla) [reply deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:68]: Uh no no, let me tag the master branch for a 2.1.0.Alpha. Thanks @dmvolod

Comment (rigazilla) [reply deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml]: I'll tag after this merge

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -155,92 +124,177 @@ spec:
...
+      email: infinispan-dev@lists.jboss.org
+  relatedImages:
+    - name: infinispan-server
+      image: quay.io/infinispan/server:12.0.0.Dev06
+  labels:
+    operated-by: infinispan-operator
+  selector:
+    matchLabels:
+      operated-by: infinispan-operator
+  links:
```
Comment (dmvolod) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml:289]: @rigazilla let's add link to the Infinispan ZulipChat

**Comment by rigazilla** [comment]
@dmvolod wdyt? it ok to merge?

**Comment by dmvolod** [comment]
Yes, LGTM, thanks, @rigazilla

**Review by dmvolod — APPROVED** [review]

**Pull request merged**